docs/sdk/clients.md

```diff
@@ -97,6 +97,11 @@ gaiacli send \
 The `--amount` flag accepts the format `--amount=<value|coin_name>`.
 :::
 
+::: tip Note
+You may want to cap the maximum gas that can be consumed by the transaction via the `--gas` flag.
+If set to 0, the gas limit will be automatically estimated.
```

Comment:
Don't we need to add the `--gas-adjustement` flag as well ?

Reply:
Done